releasenotes/notes/40142.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+releaseNotes:
+- |
+  **Updated** the load balancing algorithm used for consistent hash to MAGLEV inplance of RING_HASH.
+  This is expected to perform better with low host lookup times and also handles changes to backend
```

Comment:
'The downside of Maglev is that it is not as stable as ring hash.' seems to contradict that it handles changes better?

Reply:
It seems a doc issue. 
"With that said, for many applications including Redis, Maglev is very likely a superior drop in replacement for ring hash."
and the bench marks shows the diff and this https://docs.google.com/document/d/1IU4b76AgOXijNa4sew1gfBfSiOMbZNiEt5Dhis8QpYg/edit# explains better.

Reply:
https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto#config-cluster-v3-cluster-maglevlbconfig clearly states it is intended for minimal disruption. So doc inconsistency.